lua/wikis/commons/Infobox/Weapon.lua

```diff
@@ -49,48 +49,63 @@ function Weapon:createInfobox()
 		},
 		Center{children = {args.caption}},
 		Title{children = (args.informationType or 'Weapon') .. ' Information'},
+		Cell{name = 'Other', children = {args['other names']}},
```

Comment:
Not sure I like the input, doesn't follow our standards

Reply:
could bot convert to a param name of your chosing^^